lib/menu.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Menu do
+  def game_types do
+    display_game_types
+
+    get_value_betweenf(0, Enum.count(games) - 1)
+    |> get_players
+  end
+
+  defp display_game_types do
+    IO.puts Enum.map(games(), fn({game_type, _}) -> game_type <> "\n" end)
```

Comment:
and you could probably extract the anonymous function into a real function and just write
games
|> Enum.map(to_type/1)
|> Enum.join("\n")
|> IO.puts